### ISSUE TITLE
Termination message should appear in pod's page

### ISSUE THREAD

**Issue by tan-tan-kanarek** [kubernetes/dashboard#3942]
##### What would you like to be added
I want termination message to be visible in the dashboard.

##### Why is this needed
Although I can get it using kubectl, I want that message to be exposed to other functions in the organization that use only the dashboard.

##### Comments
I created init containers that validate configuration and logs the error to /dev/termination-log and it helps me to know what's wrong with the config, I would like to expose it to other users that don't have kubectl access.

**Comment by floreks** [comment]
I am not sure, I fully understand. Do you want to see a message why pod was killed/stopped or do you want to access some log file that was stored inside of a pod?

**Comment by tan-tan-kanarek** [comment]
I would like to see the message without dividing into the logs.

**Comment by floreks** [comment]
Is there a way to do that using `kubectl` other than diving into the container with `kubectl exec`? I am asking because we want to rely mostly on the API. If this message is only stored as part of `termination-log` file then it might not be easy to handle that.

**Comment by tan-tan-kanarek** [comment]
Yes, once I log my message to the termination log it appears in kubectl describe pod as message property under termination cause.

**Comment by floreks** [comment]
Ok, this should be easy to replicate then.
